framework/Shell/Actions/TAppHelpAction.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * TAppHelpAction class file
+ *
+ * @author Brad Anderson <belisoful@icloud.com>
+ * @link https://github.com/pradosoft/prado
+ * @license https://github.com/pradosoft/prado/blob/master/LICENSE
+ * @package Prado\Shell\Actions
+ */
+
+namespace Prado\Shell\Actions;
+
+use Prado\Shell\TShellAppAction;
+
+/**
+ * The Application specific action for help.
+ *
+ * @author Brad Anderson <belisoful[at]icloud[dot]com>
+ * @package Prado\Shell\Actions
+ * @since 4.2.0
+ */
+class TAppHelpAction extends TShellAppAction
+{
+	protected $action = 'help';
+	protected $parameters = [];
+	protected $optional = [];
+	protected $description = 'outputs the CLI Application help';
+	
+	/**
+	 * @param array $args parameters
+	 * @return bool
+	 */
+	public function performAction($args)
+	{
+	}
```

Comment:
I don't fully get what this is meant to do, is it just a stub to be used by cli applications?

Reply:
This is a stub, the actual display of help happens in the TShellInterpreter::printHelp through TAppAction when no valid option is found.  This apparently doesn't need to display the help, as it is automatically displayed when help is triggered for the application.

Reply:
I added return false, and documented that behavior